blog/_posts/2017-08-14-with-love-team-bundledore-india.md

```diff
@@ -0,0 +1,77 @@
+title: With love, Team Bundledore, India !
```

Comment:
Please follow the markdown templete for blog posts. Document details should go like this:

"--- <details - title, layout, categories etc> ---"
Also, note the syntax for categories over here. Check out the other blog posts for the example and run the site locally to avoid any confusions. :)